wp-content/plugins/jetpack/_inc/build/crowdsignal-survey.min.js

```diff
@@ -0,0 +1 @@
+!function(e,t,d){if(!e.getElementById(d)){var n,r=e.createElement(t);r.id=d,r.src="https://polldaddy.com/survey.js",(n=e.getElementsByTagName(t)[0]).parentNode.insertBefore(r,n)}}(document,"script","pd-embed");
```

Comment:
:warning: **Warning( severity 5 )**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).